### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by AlexGilleran** [TerriaJS/magda#216]
So the good news is that CKAN doesn't meet anything either: https://trello.com/c/MlQ8tdBn/56-accessibility-make-ckan-wcag-20-aa-compliant

However magda still has a couple of accessibility issues that CKAN doesn't, we should go through and make sure it works ok with a screen reader even if we're not pushing for a WCAG level yet.

E.g. right now on the front page the featured dataset links aren't focusable so a screen reader skips over them.

**Comment by chloeleichen** [comment]
I finished the first round fixes of obvious issues, more coming

**Comment by chloeleichen** [comment]
At some stage we should pay for an agency to do some accessibility audit for us, or ask Liz to do it maybe?
